src/main/java/com/tananaev/jsonpatch/operation/AddOperation.java

```diff
@@ -26,9 +26,8 @@ public String getOperationName() {
 
     @Override
     public JsonElement apply(JsonElement original) {
```

Comment:
I think it would be appropriate to rename parameter from `original` to something else. Applies to other methods as well.